examples/websockets/assets/script.js

```diff
@@ -1,4 +1,4 @@
-const socket = new WebSocket('ws://localhost:3000/ws');
+socket = new WebSocket('ws://localhost:3000/ws');
```

Comment:
Why did you remove the `const`?

Reply:
JS always conflicts with common sens in my head, calling a .close() on a const object just felt wrong. But you are correct, it should be const there.

Reply:
> JS always conflicts with common sens in my head

yeah same 😅